__tests__/app.test.js

```diff
@@ -290,6 +290,42 @@ test('should return an article by its id', () => {
     })
 })
 
+test("200: increments the votes property by the given amount", () => {
+  return request(app)
+    .patch("/api/articles/1")
+    .send({ inc_votes: 1 })
+    .expect(200)
+    .then(({ body }) => {
+      expect(body.article).toEqual(
+        expect.objectContaining({
+          article_id: 1,
+          votes: expect.any(Number),
+        })
+      );
```

Comment:
We should be expecting the whole article object instead of just a couple properties